fgpyo/sam/__init__.py

```diff
@@ -219,8 +219,8 @@ def from_path(cls, path: Union[Path, str]) -> "SamFileType":
 
 
 def _pysam_open(
-    path: SamPath, open_for_reading: bool, file_type: Optional[SamFileType] = None, **kwargs: Any
-) -> SamFile:
+    path: SamPath, open_for_reading: bool, file_type: Optional[SamFileType] = None, 
+    unmapped: bool = False, **kwargs: Any) -> SamFile:
```

Comment:
don't forget to add `unmapped` to the docstring.

Reply:
Yes, I forgot that. I'll add a test as well.